### ISSUE TITLE
Every one I add to Budibase gets a promotional email from joe@budibase.com

### ISSUE THREAD

**Issue by PaulMcF1987** [Budibase/budibase#6520]
I have added my customers to my budibase app to give them access to track all of their jobs.

They have all reported receiving the same email and I ran a trial account set up for myself and got the same email

![image](https://user-images.githubusercontent.com/75368133/176892463-a14c34ae-c187-4343-93dc-e4ed8317b5cc.png)

Why is this email sent out? Can I stop this email being sent?

**Comment by danielbolt** [comment]
I also noticed this behaviour....

**Comment by PaulMcF1987** [comment]
And now I have received reports of another email....

![image](https://user-images.githubusercontent.com/75368133/177004098-09d6ecfa-b139-4471-bb3c-14bf4c65ab3b.png)

When I add users, I am adding them as users of my apps I do not expect my users to be spammed with emails of this nature.
As far as they are concerned, they have signed up for my app, no where did I see an option to consent to budibase spamming my customers.
I need this to stop immediately.

**Comment by joebudi** [comment]
Hey Paul

We hope to resolve this on Monday.

**Comment by PaulMcF1987** [comment]
@joebudi 
Does this mean that the apps that we create using Budibase are not private? Does the Budibase team have access to view our users, databases etc?

**Comment by shogunpurple** [comment]
@PaulMcF1987 no - I’m not sure how you would come to that conclusion. We do **not** have access to the details of your users, databases and more in your self hosted instance

**Comment by PaulMcF1987** [comment]
@shogunpurple I came to that conclusion because the user profiles are saved on CouchDB and not our own DBs and the emails are sent from `joe@budibase...`
So Id agree that our apps, if using our own database would be private but it is the couchDB side of things that now concerns me since seeing all of my users emailed from joe.

**Comment by shogunpurple** [comment]
CouchDB is completely free and open source - since you are self hosted you are running your own internal instance of couch as part of the docker compose setup - it is yours. You can verify this by viewing your running containers. It has nothing to do with our cloud.

The marketing emails are sent through automation - where we email the users from a 3rd party tool. As mentioned - we will be updating that to only send to admins going forward, as to reduce the amount of emails sent.

**Comment by PaulMcF1987** [comment]
Will there be a way to opt out of emails all together, so that no automation emails are sent?

**Comment by PaulMcF1987** [comment]
and another unwanted item has appeared.... I tried the email onboarding and sent an email to my own email address to make sure the email onvarding works ok...
Once I clicked the link to set up my password I seen this

![image](https://user-images.githubusercontent.com/75368133/177006855-bbc16c94-04ed-469c-aca5-7caa1dc7916d.png)

Joe popping up again

**Comment by PaulMcF1987** [comment]
@shogunpurple any update on this fix? Joe has popped up again

![image](https://user-images.githubusercontent.com/75368133/177153741-1a58f062-0d9e-4489-b2ad-8974dfbfccb8.png)

**Comment by shogunpurple** [comment]
@PaulMcF1987 should be resolved if you update your instance - only admins should get emails now

**Comment by PaulMcF1987** [comment]
@shogunpurple how do I set it up so that no one gets emails?

**Comment by shogunpurple** [comment]
<img width="433" alt="Screenshot 2022-07-04 at 15 49 00" src="https://user-images.githubusercontent.com/11256663/177178002-f19c0c8d-3550-4c83-8312-6c6b91eb9757.png">

**Comment by PaulMcF1987** [comment]
@shogunpurple I don't mean to sound arsey, but we shouldn't need to unsubscribe from something that we didn't realise we had signed up for in the fist place.
I have used various other low code app tools via docker as well as a number of other docker hosted software and never have any of them felt the need to spam me or my users with emails.

Is there somewhere in the documentation that we should have read before hand that alerts us to the fact that there will be numerous emails sent out? And that we risk our customers (if signed up) being advertised to?
